### ISSUE TITLE
InitialOutputs is global

### ISSUE THREAD

**Issue by wilhelmy** [raboof/notion#212]
https://github.com/raboof/notion/blob/0ca62ae91a7102f9f0e379a47a7632601c72e25e/mod_xrandr/cfg_xrandr.lua#L5

Is this intentional, or can we make it local?

**Comment by wilhelmy** [comment]
Seems there are just a lot of missing `local`s in this file

**Comment by raboof** [comment]
Yes, I don't think I was aware of the repercussions of missing `local` when I wrote that ;)

**Comment by wilhelmy** [comment]
Since I brought it up I can take care of this if you want to work on other stuff regarding notion 4.
